Readme.md

```diff
@@ -89,3 +65,49 @@ See [LICENSE][here_license] for full license details.
 [here_source]: source/
 [here_image]: image/
 
+## Get started
```

Comment:
please keep the existing readme just as it is so that we can easily track the upstream wirepas repo.
move the documentation to a to-be-created application folder ("meeting_room_monitor") ?

Reply:
Done.
And moved to https://github.com/GCX-EMBEDDED/wirepas-wm-sdk/pull/5